Makefile

```diff
@@ -146,9 +146,12 @@ artifacts: docker
 	@echo 'To be added'
 
 pilot/platform/kube/config:
+	touch $@
+
+kubelink:
 	ln -fs ~/.kube/config pilot/platform/kube/
 
-.PHONY: artifacts build checkvars clean docker test setup push
+.PHONY: artifacts build checkvars clean docker test setup push kubelink
```

Comment:
We depend on this file everywhere (presubmit, circleci, etc.). I am not sure if this change is enough..

Reply:
To clarify: are you saying that there are other build entrypoints that this change would break, because they depend on the existing symlinking behavior?

If so, could you weigh in on my first approach here: https://github.com/istio/istio/pull/1950? I believe that first attempt would also smooth over the first time build experience, while maintaining backwards-compatibility with the current behavior.

Reply:
Okay, I've updated the `.circleci/config.yml` as well to invoke the new make target. Another option is to assume that anything that invokes `setup` intends to create the symlink, and to update that target rather than introducing a new one, but I'm not sure how safe of an assumption that is.

I'll see if that addition appeases the presubmit tests.